### ISSUE TITLE
Palette items should drop VI contents

### ISSUE THREAD

**Issue by buckd** [ni/niveristand-engine-simulation-toolkit-fpga-ip#5]
Basically, the only way to get started with the Engine Simulation Toolkit is to start from one of the examples. The problem with this is you first have to know the examples are there and then be able to find them.

The palette items drop VIs that are used for the toolkit, but all of them need other supporting items around them (memories, SCTLs, prerequisite initialize VIs, etc.). Instead of putting all of this on the user, the palette items should drop VI contents. For instance, if I need a new APU to go across multiple FPGAs, I should be able to drop a new APU item that includes the Sync Trigger setup, the bus initialization and the processing loop all inside the SCTLs that must be around them.

Basically, this would provide a better solution to the copy/paste from the examples that every user has to do anyway.
